jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants.properties

```diff
@@ -94,5 +94,13 @@ InvalidPom=Corrupted file content
 UploadFailed=Upload of service task failed
 InternalUploadError=Unable to add service task due to internal error
 AddTaskSuccess=Service tasks successfully added - 
+SkippedMessage=The following service tasks have been skipped as the repository already contains services with the same name as: {0}.
+RemoveTaskSuccess=Service task {0} successfully removed.
 Installing=Installing service task...
-InstallServiceTaskParams=Required parameters for service task
\ No newline at end of file
+InstallServiceTaskParams=Required parameters for service task
+RemoveServiceTaskAction=Remove
```

Comment:
Imo here needs to be 'Delete' ... there seems to be consistency across workbench using the Delete whenever removing/deleting something.

Reply:
done